Mathlib/Topology/MetricSpace/Baire.lean

```diff
@@ -54,78 +54,73 @@ Two versions of these theorems are given.
 The first states that complete pseudo_emetric spaces are Baire. -/
 instance (priority := 100) baire_category_theorem_emetric_complete [PseudoEMetricSpace α]
     [CompleteSpace α] : BaireSpace α := by
-  refine' ⟨fun f ho hd => _⟩
-  let B : ℕ → ℝ≥0∞ := fun n => 1 / 2 ^ n
+  refine' ⟨fun f ho hd ↦ _⟩
+  let B : ℕ → ℝ≥0∞ := fun n ↦ 1 / 2 ^ n
   have Bpos : ∀ n, 0 < B n := by
     intro n
-    simp only [B, one_div, one_mul, ENNReal.inv_pos]
+    simp only [one_div, one_mul, ENNReal.inv_pos]
     exact pow_ne_top two_ne_top
   /- Translate the density assumption into two functions `center` and `radius` associating
     to any n, x, δ, δpos a center and a positive radius such that
-    `closed_ball center radius` is included both in `f n` and in `closed_ball x δ`.
+    `closedBall center radius` is included both in `f n` and in `closedBall x δ`.
     We can also require `radius ≤ (1/2)^(n+1)`, to ensure we get a Cauchy sequence later. -/
-  have : ∀ n x δ, δ ≠ 0 → ∃ y r, 0 < r ∧ r ≤ B (n + 1) ∧ closed_ball y r ⊆ closed_ball x δ ∩ f n :=
+  have : ∀ n x δ, δ ≠ 0 → ∃ y r, 0 < r ∧ r ≤ B (n + 1) ∧ closedBall y r ⊆ closedBall x δ ∩ f n :=
     by
     intro n x δ δpos
     have : x ∈ closure (f n) := hd n x
     rcases EMetric.mem_closure_iff.1 this (δ / 2) (ENNReal.half_pos δpos) with ⟨y, ys, xy⟩
     rw [edist_comm] at xy
-    obtain ⟨r, rpos, hr⟩ : ∃ r > 0, closed_ball y r ⊆ f n :=
+    obtain ⟨r, rpos, hr⟩ : ∃ r > 0, closedBall y r ⊆ f n :=
       nhds_basis_closed_eball.mem_iff.1 (isOpen_iff_mem_nhds.1 (ho n) y ys)
-    refine' ⟨y, min (min (δ / 2) r) (B (n + 1)), _, _, fun z hz => ⟨_, _⟩⟩
+    refine' ⟨y, min (min (δ / 2) r) (B (n + 1)), _, _, fun z hz ↦ ⟨_, _⟩⟩
     show 0 < min (min (δ / 2) r) (B (n + 1))
     exact lt_min (lt_min (ENNReal.half_pos δpos) rpos) (Bpos (n + 1))
     show min (min (δ / 2) r) (B (n + 1)) ≤ B (n + 1)
     exact min_le_right _ _
-    show z ∈ closed_ball x δ
+    show z ∈ closedBall x δ
     exact
       calc
         edist z x ≤ edist z y + edist y x := edist_triangle _ _ _
         _ ≤ min (min (δ / 2) r) (B (n + 1)) + δ / 2 := (add_le_add hz (le_of_lt xy))
         _ ≤ δ / 2 + δ / 2 := (add_le_add (le_trans (min_le_left _ _) (min_le_left _ _)) le_rfl)
         _ = δ := ENNReal.add_halves δ
-        
     show z ∈ f n
-    exact
-      hr
-        (calc
-          edist z y ≤ min (min (δ / 2) r) (B (n + 1)) := hz
-          _ ≤ r := le_trans (min_le_left _ _) (min_le_right _ _)
-          )
+    exact hr (calc
+      edist z y ≤ min (min (δ / 2) r) (B (n + 1)) := hz
+      _ ≤ r := le_trans (min_le_left _ _) (min_le_right _ _))
   choose! center radius Hpos HB Hball using this
-  refine' fun x => (mem_closure_iff_nhds_basis nhds_basis_closed_eball).2 fun ε εpos => _
+  refine' fun x ↦ (mem_closure_iff_nhds_basis nhds_basis_closed_eball).2 fun ε εpos ↦ _
```

Comment:
I think it would be better to leave these as `=>`, as it keeps the diff much smaller and makes forward-porting easier.

We can always upgrade them all after the port.

Reply:
I learned about changing `=>` to `↦` on the porting wiki. Should the porting wiki be updated?

Reply:
In my opinion yes. If we really want to make this substitution we should have a script to do it automatically